src/simulation/openables/BronzeHAMChest.ts

```diff
@@ -0,0 +1,25 @@
+import LootTable from '../../structures/LootTable';
+import SimpleOpenable from '../../structures/SimpleOpenable';
+import { rand } from '../../util';
+
+const rolls = rand(1, 5);
+
+/* Gem loots */
+const GemTable = new LootTable({ limit: 100 })
+	.add('Uncut sapphire', 1, 25)
+	.add('Sapphire', 1, 25)
+	.add('Uncut emerald', 1, 15)
+	.add('Emerald', 1, 15)
+	.add('Uncut ruby', 1, 5)
+	.add('Ruby', 1, 5)
+	.add('Uncut diamond', 1, 5)
+	.add('Diamond', 1, 5);
+
+const BronzeHAMChestTable = new LootTable().every('Coins', [1, 400]).every(GemTable, rolls);
+
+export default new SimpleOpenable({
```

Comment:
the wiki doesnt say anything about the different chests having different loot tables.
combine all the different chests into one table and then place a `TODO` in it so that we can check back occasionally to see if the wiki has exact rates

Reply:
Adding a TODO and leaving them slight different for a more interactive user experience since neither is correct.